### PR TITLE
These OIDs are used in X.509 certificates issued in Russia.

### PR DESCRIPTION
**Pull request by beldmit**
Here are some OIDs used in Russian X.509 certificates.

OBJ_OGRNIP denotes a specific legal status of the certificate owner.
OBJ_classSignTool* denotes a level of certification of the software created the certificate.

http://www.garant.ru/products/ipo/prime/doc/70033464/ is the relevant link (in Russian)

### CODE REVIEW

File: crypto/objects/objects.txt [hunk 1]
```diff
@@ -1391,8 +1391,16 @@ id-tc26-gost-28147-constants 1	: id-tc26-gost-28147-param-Z : GOST 28147-89 TC26
 member-body 643 3 131 1 1	: INN	: INN
 member-body 643 100 1		: OGRN	: OGRN
 member-body 643 100 3		: SNILS	: SNILS
+member-body 643 100 5		: OGRNIP	: OGRNIP
 member-body 643 100 111	: subjectSignTool	: Signing Tool of Subject
 member-body 643 100 112	: issuerSignTool	: Signing Tool of Issuer
+member-body 643 100 113  : classSignTool   : Class of Signing Tool
```
Comment (mattcaswell) [inline crypto/objects/objects.txt]: Is there a reference to some standard we could add as a comment here to explain where these come from?

Comment (beldmit) [reply crypto/objects/objects.txt]: Well, the most relevant link seems to be http://www.garant.ru/products/ipo/prime/doc/70033464/

Comment (mattcaswell) [reply crypto/objects/objects.txt]: Is that link likely to be stable? If so add that - if not perhaps just a sentence or so on what these OIDS are for.

Comment (beldmit) [reply crypto/objects/objects.txt]: Added to the pull request. It should be rather stable.

Comment (mattcaswell) [reply crypto/objects/objects.txt]: Did you push? I don't see it?

Comment (beldmit) [reply crypto/objects/objects.txt]: Well. Got totally messed.
The ```git commit --amend``` does weird things. And when I try to checkout a new branch, it contains OCSP fixes, so I have no ideas.

Comment (beldmit) [reply crypto/objects/objects.txt]: See #8309

**Pull request closed without merging**